src/components/periodic-report/dto/periodic-report.dto.ts

```diff
@@ -50,7 +51,7 @@ class PeriodicReport extends Resource {
   @Field()
   readonly receivedDate: SecuredDateNullable;
 
-  readonly reportFile: DefinedFile;
+  readonly directory: DefinedFile;
```

Comment:
A DB migration class needs to be written to:
- Move `ProgressReport.reportFile` to `pnp`
- Create directories
- Move other report files into the directory